src/lib/rate_control/CMakeLists.txt

```diff
@@ -32,11 +32,11 @@
 ############################################################################
 
 px4_add_library(RateControl
-	RateControl.cpp
-	RateControl.hpp
+	rate_control.cpp
```

Comment:
Just because file name of most of the shared file names were lower case. (Since I moved the file into the shared lib and outside of the mc_rate_control). The class is still camel cased as is with other libraries. I can change it if we want to preserve the previous file name

Reply:
@sfuhrer @Jaeyoung-Lim - also -  https://google.github.io/styleguide/cppguide.html#File_Names, if we are going to try and stick to the new guide